src/components/FileUploaderField/index.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import FormField from '../FormField';
+import FileUploader from '../FileUploader';
+
+const FileUploaderField = React.forwardRef(
+  ({ label, helper, inputRef, ...props }, ref) => (
+    <FormField label={label} helper={helper} {...props} ref={ref}>
+      <FileUploader ref={inputRef} {...props} />
+    </FormField>
+  ),
+);
+
+FileUploaderField.propTypes = {
+  helper: PropTypes.node,
+  id: PropTypes.string.isRequired,
+  label: PropTypes.node,
+  inputRef: PropTypes.shape({
+    current: PropTypes.instanceOf(Element),
+  }),
+};
+
+FileUploaderField.displayName = 'FileUploaderField';
```

Comment:
`displayName` is not needed anymore IIRC 🙂

Reply:
dammit I keep forgetting 😄

Reply:
I checked and we need it here since it's not an SC but a normal component

Reply:
Oh yeah of course 🤦